GLTFSDK/GenerateSchemaJsonHeader.ps1

```diff
@@ -225,7 +225,7 @@ Get-ChildItem -File -Recurse -Filter "*.json" "$SCHEMAS_PATH" | Sort-Object | Fo
    AppendToHeader $varText $globalIndentLevel
 }
 
-DeclareGltfSchemaVars $CLASS_NAME $mapNamesList
+DeclareGltfSchemaVars $CLASS_NAME
```

Comment:
It doesn't look like the first parameter is used either. Maybe remove both of them?
```suggestion
DeclareGltfSchemaVars
```

Reply:
Sure